CHANGELOG.md

```diff
@@ -59,11 +60,13 @@
 * [BUGFIX] Fixed cache fetch error on Redis Cluster. #4056
 * [BUGFIX] Ingester: fix issue where runtime limits erroneously override default limits. #4246
 * [BUGFIX] Ruler: fix startup in single-binary mode when the new `ruler_storage` is used. #4252
+<<<<<<< HEAD
```

Comment:
Bit of merge conflict debris to clean up

Reply:
Sorry for that :D